app/models/user.rb

```diff
@@ -346,6 +346,17 @@ def volunteer_duties(conference)
     events.where(program_id: conference.program.id, 'event_users.event_role': 'volunteer')
   end
 
+  def count_registration_tickets(conference)
+    count = 0
+    ticket_purchases.by_conference(conference).each do |ticket_purchase|
+      if ticket_purchase.ticket.registration_ticket
+        count += 1
+      end
+    end
```

Comment:
Technically this can be compressed into [a reduce method](https://medium.com/@james.a.hughes/using-the-reduce-method-in-ruby-907f3c18ae1f), but given the internal check, it's probably fine.

Reply:
So I rewrote it, but it doesn't look simpler than the for loop... I think I'll keep it as it is. Thanks for bringing it up tho. 

```
ticket_purchases.by_conference(conference).inject do |count, ticket_purchase|
  if ticket_purchase.ticket.registration_ticket
    count += 1
  end
end